mlflow/azureml/__init__.py

```diff
@@ -611,7 +611,12 @@ def _create_mlflow_wheel(mlflow_dir, out_dir):
 
 from azureml.core.model import Model
 from mlflow.pyfunc import load_model
-from mlflow.pyfunc.scoring_server import parse_json_input, _get_jsonable_obj
+from mlflow.pyfunc.scoring_server import parse_json_input
+
+try:
+    from mlflow.utils.proto_json_utils import _get_jsonable_obj
+except ImportError:  # Back Compat for https://github.com/mlflow/mlflow/pull/3687
+    from mlflow.pyfunc.scoring_server import _get_jsonable_obj
```

Comment:
Is the concern here that we might deploy an MLflow model to AzureML and install an older version of MLflow when creating the serving container Docker image? The deployment procedure seems to ensure that we're pulling in the latest MLflow version (https://github.com/mlflow/mlflow/blob/aaed8043dd9d206a8090a480493118be9ba4240e/mlflow/azureml/__init__.py#L554-L561), so I think it's okay to remove the try/except here.

Let me know if I'm missing something here.